js/src/components/google-combo-account-card/account-card-description/account-card-description.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import AccountDetails from './account-details';
+import CreatingAccounts from './creating-accounts';
+import useAccountsData from '.~/hooks/useAccountsData';
+
+/**
+ * AccountCardDescription component.
+ * @return {JSX.Element} AccountCardDescription component.
+ */
+const AccountCardDescription = () => {
+	const { creatingWhich, google, googleAdsAccount, googleMCAccount } =
+		useAccountsData();
+
+	return (
+		<>
+			{ creatingWhich ? (
```

Comment:
@ankitrox Let's return `CreatingAccounts` early if `creatingWhich` is `true`
```suggestion
if ( creatingWhich ) {
	return <CreatingAccounts creatingAccounts={ creatingWhich } />;
}

return (
	<AccountDetails
		email={ google.email }
		googleAdsID={ googleAdsAccount.id }
		googleMerchantCenterID={ googleMCAccount.id }
	/>
);
```

Reply:
@asvinb Isn't it the same thing right now?

Reply:
@ankitrox Technically it's the same thing. The early return version is easier to read and cleaner IMHO 😁